src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -597,4 +657,13 @@ protected void outputException(X x, HttpServletResponse resp, Format format, Pri
       // TODO
     }
   }
+
+  public String getPropertyName(String field) {
```

Comment:
This won't work for count since it doesn't necessarily have a field in it .. not sure if we are expecting be COUNT() or COUNT(*)?